frictionless/formats/parquet/parser.py

```diff
@@ -25,7 +25,10 @@ class ParquetParser(Parser):
 
     def read_cell_stream_create(self):
         control = ParquetControl.from_dialect(self.resource.dialect)
-        file = platform.fastparquet.ParquetFile(self.resource.normpath)
+        handles = platform.pandas.io.common.get_handle(
```

Comment:
@shashigharti 
Will it be the same performance-wise for local files? Or it's better to use this only for remote ones? I think, currently, it will create handles for both local/remote

Reply:
I agree! I will make changes to create handle only for remote files. Thank you